config/locales/es/budgets.yml

```diff
@@ -58,6 +58,7 @@ es:
         map_location: "Ubicación en el mapa"
         map_location_instructions: "Navega por el mapa hasta la ubicación y coloca el marcador."
         map_remove_marker: "Eliminar el marcador"
+        location: "Información adicional de la localización"
```

Comment:
Maybe to be consistent with other spanish texts around locations the word **localización** should be changed for **ubicación** as that's what the map uses 3 times on title/subtitle/checkbox

Reply:
Good catch!